### DISCUSSION TITLE
DMCA takedown is slow

### DISCUSSION THREAD

**Discussion by xAstroBoy** [community/community#23082] [New to GitHub]
<p>I filed 3 dmca takedowns for 2 repos and 3 forks, i want them all down, because is my private work, but github yet hasn’t taken it down after 3 days…</p>

**Comment by jfullstackdev** [comment]
<p>how did they get the code if it’s private in the first place ?</p>
<p>hopefully, moderators will see this too and take a look at your request</p>

**Comment by xAstroBoy** [comment]
<p>They are taking down all these leaks as we speak, they managed to steal the drive url that very few people had (we suspect a leaker) and turned the dll back into a project.</p>

**Comment by xAstroBoy** [comment]
<p>Hell, they even tried to circumvent the github DMCA by rehosting into a private server and open another github repo with the links.</p>
<p>there’s one fork left of it, hopefully github takes that repo down as well.</p>

**Comment by xAstroBoy** [comment]
<p>Annnd he is trying  to workaround against the DMCA for the 4th time now…</p>

**Comment by xAstroBoy** [comment]
<p>Bruh, he got banned and this is the second account, doing the same thing.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there,</p>
<p>First of all, I am so sorry to hear you’ve had this experience and you felt the process did not move as quickly as you would have liked. The GitHub Community Forum is unable to assist in this matter, I recommend continuing to communicate on the takedown request as they are best-equipped to assist. Best of luck!</p>

**Comment by xAstroBoy** [comment]
<p>it has been handled, thanks anyways &lt;3</p>
